packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,10 +7,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [37.0.0]
+
+### Added
+
+- **BREAKING:** `TokenBalancesControllerMessenger` must allow the `AssetsContractController:getERC20BalanceOf` action in addition to its previous allowed actions ([#4397](https://github.com/MetaMask/core/pull/4397))
```

Comment:
Hmm, additions are not typically breaking. This seems more like a change.

"Added" is for new features/capabilities/enhancements, whereas these are changes to how the package must be used.

Reply:
There was a discussion with @mcmire about this in a previous release candidate: 

> The reason it's a breaking change is that consumers now needs to allowlist that action or event when initializing the messenger, otherwise an error will be thrown when attempting to call or subscribe to that action or event.
> https://github.com/MetaMask/core/pull/4498#discussion_r1664531980

However, regardless of the above, our messengers currently behave like this isn't a breaking change. These are tickets for fixing this behavior:
- https://github.com/MetaMask/core/issues/4414
- https://github.com/MetaMask/core/issues/4501

Reply:
It seems like @Gudahtt is arguing not that this should not be a breaking change, but that this should not be in Added, but instead in Changed.

I am okay with moving this to Changed.

Reply:
But I think moving this entry to the "Changed" heading makes sense, given that this alters the messenger's behavior rather expand its API or capabilities.

Reply:
Fixed here https://github.com/MetaMask/core/pull/4548/commits/225ed0f97d08d82d7f3f76d4c45f78d4a5c33349

I also moved the entry below this one.